docs/user_guide/1 - Installing.ipynb

```diff
@@ -38,9 +45,24 @@
    "metadata": {},
    "source": [
     "```bash\n",
-    "cp nbextensions/* $(ipython locate)/nbextensions/\n",
+    "python -m nbgrader --help\n",
```

Comment:
It might be nice to actually make this a code cell, so the output gets displayed in the docs. You can use the `run` command from `nbgrader.utils` so that the doc checker machinery will catch any errors if it fails to run.